workorders.lic

```diff
@@ -286,28 +286,63 @@ class WorkOrders
       stock_needed = 1
       order_rem(info['stock-room'], stock_needed, info['stock-number'])
     when 'You get'
-      /are (.*) parts left of the water/ =~ bput('count my water', 'are .* parts left of the water')
+      /are (.*) parts left of the water|only (.*) part left/ =~ bput('count my water', 'are .* parts left of the water')
```

Comment:
Okay, I will tomorrow.

If you want, you can reject the request, unless that is not how it 
works. I will make changes tomorrow and test again.

-- 
Regards,
Dr. Walter Wesley "Wes" Snyder V
484-300-4833 (h)
484-431-3355 (m)
484-302-7006 (f)

On 09/04/2016 11:03 PM, Robert Herbig wrote:
>
> In workorders.lic 
> <https://github.com/rpherbig/dr-scripts/pull/737#discussion_r77466470>:
>
> > @@ -286,28 +286,63 @@ class WorkOrders
> >        stock_needed = 1
> >        order_rem(info['stock-room'], stock_needed, info['stock-number'])
> >      when 'You get'
> > -      /are (.*) parts left of the water/ =~ bput('count my water', 'are .* parts left of the water')
> > +      /are (.*) parts left of the water|only (.*) part left/ =~ bput('count my water', 'are .* parts left of the water')
>
> I'd prefer you to make these changes if possible. You know the code 
> better and what needs to be tested to have confidence in it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/rpherbig/dr-scripts/pull/737/files/7aaafb3fc3d1daa463a35a97157ca897a5e00fd4#r77466470>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AU8XXPtt78vsOr-6k_XUNA05YOaaP8I3ks5qm4afgaJpZM4J0roH>.
>